src/frontend/README.md

```diff
@@ -0,0 +1,34 @@
+# React + TypeScript + Vite
```

Comment:
let's add the install/run commands in here as well please. anything we need to get started without having to look at anywhere else